### PR TITLE
Mvp nav patterns content

### PR DESCRIPTION
**Pull request by halocline**
MVP Navigation Patterns Content -

[Preview](https://deploy-preview-769--keen-mayer-a86c8b.netlify.app/templates/navigation)

Includes:
- Navigation page, which includes introductory copy, then links to the following pages:
  - Hub & Spoke Navigation
  - Persistent Navigation

- Hub & Spoke Navigation page includes 1 initial example, plus guidance and related content
- Persistent Navigation page includes examples for Header Nav and Minimal Sidebar, plus guidance and related content

### CODE REVIEW

File: aries-site/src/data/structure.js [hunk 1]
```diff
@@ -821,6 +831,56 @@ export const structure = [
...
+      'Header',
+      'Global Sidebar',
+      'Hub and Spoke Navigation',
+      'Persistent Navigation',
+      'React Router',
+    ],
+  },
+  {
+    name: 'Hub and Spoke Navigation',
+    description: `A navigation pattern promoting focussed execution through a central "hub" from which all
```
Comment (taysea) [inline aries-site/src/data/structure.js:850]:
```suggestion
    description: `A navigation pattern promoting focused execution through a central "hub" from which all
```

File: aries-site/src/pages/templates/hub-and-spoke-navigation.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    <ContentSection>
+      <Subsection name="Guidance">
+        <SubsectionText>
+          Hub &amp; Spoke Navigation works well for applications primarily
+          concerned with task execution, as opposed to presenting large volumes
+          of content.
+        </SubsectionText>
+        <SubsectionText>
+          The "hub" cleanly presents multiple discreet application sections or
+          "spokes." Each "spoke" keeps the user focussed on the task at hand and
```
Comment (taysea) [inline aries-site/src/pages/templates/hub-and-spoke-navigation.js:41]:
```suggestion
          "spokes." Each "spoke" keeps the user focused on the task at hand and
```

**Comment by britt6612** [comment]
Looks good! Wonder what images we can use for cards to tie it off

**Comment by halocline** [comment]
> Looks good! Wonder what images we can use for cards to tie it off

I had reached out to folks on Slack for help with that. @L0ZZI mentioned he would incorporate those with some others he was working on. [Convo here](https://hpe.slack.com/archives/GPPLUK6LR/p1589229817003700)

**Pull request merged**